core/AUTHORS

```diff
@@ -0,0 +1,6 @@
+Holger krekel <holger at merlinux dot eu>
+juga <juga at riseup dot net>
+azul <azul at riseup dot net>
+Daniel Kahn Gillmor <dkg at fifthhorseman dot net>
+Vincent Breitmoser <look at my dot amazin dot horse>
+Oliver Wiese <oliver dot wiese at fu-berlin dot de>
```

Comment:
did you check that all these authors have significant (say >1% contributions to the code base?)